### PR TITLE
🚧Add rule to solution editorconfig

### LINKED ISSUE #168: 🚧Add rule to solution editorconfig
### Complete The Item Below

- [X] I have updated the title without removing the 🚧 emoji.

### Description

Add a new rule to the solution-wide **.editorconfig** file to ignore warnings and suggestions for outside `using` statement placement.

**Here is the rule to add:**
```ini
# IDE0065: Misplaced using directive
csharp_using_directive_placement = inside_namespace
```

> **Note** There are two **_.editorconfig_** files in this solution.  One is in the root of the repo and it is the _**root**_ config file.  The other is in the _**Testing/**_ directory and are additional rules for the unit tests projects.  The unit testing config file inherits the rules from the solution level config file.

> **Note** This will remove any warnings in **_Visual Studio_** about moving the using statements outside of the namespaces.  _**JetBrains Rider**_ users should not get this warning.

### Acceptance Criteria

> **Note** The tasks below need to be done in the order below.

```[tasklist]
### Tasks
- [x] The solution level config updated
- [x] The testing level config updated
```

### ToDo Items

- [X] Change type labels added to this issue.  Refer to the _**Change Type Labels**_ section below.
- [X] Priority label added to this issue.  Refer to the _**Priority Type Labels**_ section below.
- [X] Issue linked to the correct project _(if applicable)_.
- [X] Issue linked to the correct milestone _(if applicable)_.
- [x] Draft pull request created and linked to this issue _(only required with code changes)_.

### Issue Dependencies

_No response_

### Related Work

_No response_

### Additional Information:

**_<details closed><summary>Change Type Labels</summary>_**

| Change Type         | Label                |
|---------------------|----------------------|
| Bug Fixes           | `🐛bug` |
| Breaking Changes    | `🧨breaking changes` |
| New Feature         | `✨new feature` |
| CICD Changes        | `♻️cicd` |
| Code Doc Changes    | `🗒️documentation/code` |
| Product Doc Changes | `📝documentation/product` |
</details>

**_<details closed><summary>Priority Type Labels</summary>_**

| Priority Type       | Label             |
|---------------------|-------------------|
| Low Priority        | `low priority`    |
| Medium Priority     | `medium priority` |
| High Priority       | `high priority`   |
</details>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### PR DESCRIPTION
**Pull request by kselena**
<h1 style="font-weight:bold" align="center">Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: feature/<issue-num>-<description>
      Example: feature/123-my-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_development_** branch with the name **_develop_**.

- [x] The **_base(target)_** branch for this pull request is a **_development_** branch with the name **_develop_**.
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

💡For more information on branching, refer to the project's [branching documentation](https://github.com/KinsonDigital/.github/blob/master/docs/BranchingFullGitFlow.md).

- [x] The associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] The pull request title matches the title of the associated issue.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] The assigned project matches the project of the associated issue.

- [x] The assigned milestone matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out, and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_develop_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: Testing/.editorconfig [hunk 1]
```diff
@@ -19,6 +19,10 @@ charset = utf-8-bom
 # Expression-bodied members
 csharp_style_expression_bodied_methods = true:none
 
+# Ignore warnings and suggestions for outside using statement placement
+dotnet_style_qualification_for_field = false:suggestion
+
+
```
Comment (KinsonDigitalAdmin) [inline Testing/.editorconfig:25]: Remove this rule from this config.  This will automatically inherit the newly added rule from the solution-wide config.  This is why you do not need to add it in both.

This info is all in the issue description.
image



```suggestion
```

File: .editorconfig [hunk 2]
```diff
@@ -20,6 +20,9 @@ charset = utf-8-bom
 # Organize usings
 dotnet_sort_system_directives_first = true
 
+# Ignore warnings and suggestions for outside using statement placement
+dotnet_style_qualification_for_field = false:suggestion
```
Comment (KinsonDigitalAdmin) [inline .editorconfig:24]: Fix this rule.  You added the incorrect rule to the file.  The rule to add is in the issue description.

```suggestion
# IDE0065: Misplaced using directive
csharp_using_directive_placement = inside_namespace
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

**Review by CalvinWilkinson — CHANGES_REQUESTED** [review]
make following changes

**Review by KinsonDigitalAdmin — APPROVED** [review]

**Review by KinsonDigitalAdmin — DISMISSED** [review]

**Review by CalvinWilkinson — DISMISSED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]
